packages/expansion-panel/src/components/expansion-panel-head/index.tsx

```diff
@@ -0,0 +1,5 @@
+import { FC } from 'react';
+
+export const ExpansionPanelHead: FC<any> = ({ children }) => children;
```

Comment:
Bare for at props er litt mer ryddig:

```suggestion
import type { FC, PropsWithChildren } from 'react';

interface Props {}

export const ExpansionPanelHead: FC<PropsWithChildren<Props>> = ({ children }) => children;
```